test/io_test.dart

```diff
@@ -9,146 +9,126 @@ import 'dart:io';
 
 import 'package:dart_style/src/io.dart';
 import 'package:path/path.dart' as p;
-import 'package:scheduled_test/descriptor.dart' as d;
-import 'package:scheduled_test/scheduled_test.dart';
+import 'package:test_descriptor/test_descriptor.dart' as d;
+import 'package:test/test.dart';
 
 import 'package:dart_style/src/formatter_options.dart';
 
 import 'utils.dart';
 
 void main() {
-  setUpTestSuite();
-
   var overwriteOptions = new FormatterOptions(OutputReporter.overwrite);
 
   var followOptions =
       new FormatterOptions(OutputReporter.overwrite, followLinks: true);
 
-  test('handles directory ending in ".dart"', () {
-    d.dir('code.dart', [
+  test('handles directory ending in ".dart"', () async {
+    await d.dir('code.dart', [
       d.file('a.dart', unformattedSource),
     ]).create();
 
-    schedule(() {
-      var dir = new Directory(d.defaultRoot);
-      processDirectory(overwriteOptions, dir);
-    }, 'Run formatter.');
+    var dir = new Directory(d.sandbox);
+    processDirectory(overwriteOptions, dir);
 
-    d.dir('code.dart', [
+    await d.dir('code.dart', [
       d.file('a.dart', formattedSource),
     ]).validate();
   });
 
-  test("doesn't touch unchanged files", () {
-    d.dir('code', [
+  test("doesn't touch unchanged files", () async {
+    await d.dir('code', [
       d.file('bad.dart', unformattedSource),
       d.file('good.dart', formattedSource),
     ]).create();
 
-    modTime(String file) {
-      return new File(p.join(d.defaultRoot, 'code', file)).statSync().modified;
-    }
+    DateTime modTime(String file) =>
+        new File(p.join(d.sandbox, 'code', file)).statSync().modified;
 
-    var badBefore;
-    var goodBefore;
+    DateTime badBefore;
+    DateTime goodBefore;
 
-    schedule(() {
-      badBefore = modTime('bad.dart');
-      goodBefore = modTime('good.dart');
+    badBefore = modTime('bad.dart');
+    goodBefore = modTime('good.dart');
```

Comment:
May as well unify these with the above two lines:

```dart
var badBefore = modTile('bad.dart');
var goodBefore = modTime('good.dart');
```

Reply:
yup